bin/coreos-install

```diff
@@ -387,6 +387,11 @@ gpg --batch --quiet --import <<<"$GPG_KEY"
 echo "Downloading the signature for ${IMAGE_URL}..."
 wget --no-verbose -O "${WORKDIR}/${SIG_NAME}" "${SIG_URL}"
 
+# We are at the point of no return, so wipe disk labels that are missed below.
+# In particular, ZFS writes labels in the last half-MiB of the disk.
+dd if=/dev/zero of="${DEVICE}" bs=1024 count=512 2>/dev/null \
```

Comment:
Why not omit the `bs` flag and then you can also drop the division by two below?